app/Mcp/Tools/ListUnhealthyResources.php

```
@@ -0,0 +1,421 @@
+                [, $serviceRows] = $this->collectUnhealthyServices(
+                    $teamId,
+                    sampleOnly: false,
+                    samplePerType: 1,
+                    skip: $skip,
+                    take: $need,
+                );
+                $rows = $serviceRows->values()->all();
+                $page = array_merge($page, $rows);
+                $need -= count($rows);
+                $skip = 0;
+            }
```

PHPMD (2.15.0)

[error] The method paginateFullList uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)